controllers/application/service.go

```diff
@@ -13,6 +13,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var defaultPromPort = corev1.ServicePort{
```

Comment:
Suuuuper nitpicky but could this be renamed to `defaultPrometheusPort`? I skimped over and didn't understand what it was at first